site-src/guides/migrating-from-ingress.md

```diff
@@ -0,0 +1,261 @@
+# Migrating from Ingress
+
+The Gateway API is not backward compatible with the [Ingress API](https://kubernetes.io/docs/concepts/services-networking/ingress/). As a result, if you are an Ingress user and would like to migrate, you will need to convert your Ingress resources to Gateway API resources. 
+
+This guide will help you with the conversion. It will:
+
+* Explain the key differences between the Ingress API and the Gateway API.
+* Map the Ingress features to the Gateway API features.
+* Show an example of an Ingress resource converted to Gateway API resources.
+* Mention a helper tool for automatic conversion.
+
+At the same time, it will not prepare you for a live migration or explain how to convert some vendor-specific features of your Ingress controller. Additionally, since the Ingress API only covers HTTP/HTTPS traffic, this guide does not cover the Gateway API support for other protocols.
```

Comment:
Tiny nit
```suggestion
At the same time, it will not prepare you for a live migration or explain how to convert some implementation-specific features of your Ingress controller. Additionally, since the Ingress API only covers HTTP/HTTPS traffic, this guide does not cover the Gateway API support for other protocols.
```